src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -550,6 +762,22 @@ export function pickMethylationEnrichmentProfiles(
     });
 }
 
+export function pickAllGenericAssayEnrichmentProfiles(
```

Comment:
Are we using this function? It looks like it's not used by other functions, just wanted to double-check this.

Reply:
Yes I think it is used in the genericAssayAllEnrichmentProfilesGroupedByGenericAssayType of ComparisonStore.ts. In this case we can get all three of generic assay data grouped by generic assay data type.